src/conf_macros.m4

```diff
@@ -816,9 +816,12 @@ AC_DEFUN([WITH_SSSD_USER],
     fi
 
     AC_SUBST(SSSD_USER)
-    AC_DEFINE_UNQUOTED(SSSD_USER, "$SSSD_USER", ["The default user to run SSSD as"])
+    AC_DEFINE_UNQUOTED(SSSD_USER, "$SSSD_USER", ["Supported user to run SSSD as (besides root)"])
```

Comment:
s/Supported user to run SSSD as (besides root)/Supported user, besides root, for running SSSD

Reply:
IMO one of the below options sounds best.

`Non-root user to run SSSD as`

`Supported non-root user to run SSSD as`

Reply:
Updated per Justin's comment.